utils/RequestUtil.js

```diff
@@ -73,20 +73,23 @@ var RequestUtil = {
   json: function (options) {
     // Default assign options to empty object
     options || (options = {});
+    var requestID = options.url;
```

Comment:
Query params should effect the ID. So should the method, headers and possibly data for POST, PUT, PATCH.